maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenProjectProvider.java

```diff
@@ -747,6 +747,8 @@ private void updateDtdVersion( IProject project, String dtdVersion, String arche
                     }
 
                     editModel.save();
+                    
+                    editModel.releaseFromEdit();
```

Comment:
We need to wrap the model edit in a try catch to make sure we always releadFromEdit() and then we can remove the outer try/catch.